debug_src/debug.js

```diff
@@ -0,0 +1,78 @@
+const preact = require('../');
+
+const original = {
+	h: preact.h
+};
+
+const config = {
+	undefinedComponents: true,
+	stringRefs: true,
+	arrayChildren: true
+};
+
+preact.h = function(nodeName, attributes) {
```

Comment:
I want to check if we can prune the stack trace - we'd know it's 2 levels up, and the stack is just a String so we could remove the first two lines.  My reasoning is that `preact` is actually a frozen object when bundled via Webpack 2 - this is already causing issues with React Ho Loader since it does the patching technique used here, and I'm worried it's going to become an issue.  Thoughts?